lib/middleware/errors.js

```diff
@@ -47,8 +44,33 @@ function onUncaughtException(error) {
   setImmediate(process.exit.bind(null, 66), 1000);// eslint-disable-line no-undef
 }
 
+function renderView(err, req, res, next) {
```

Comment:
I assume that the `index.hbs` file in each of the FE apps will need to have the `{{#if errorDetails}}` conditions, right?